.github/workflows/ci.yaml

```diff
@@ -26,21 +21,16 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-          - '3.8'
-          - '3.9'
-          - '3.10'
-          - '3.11'
-          - '3.12'
+          - "3.9"
+          - "3.10"
+          - "3.11"
+          - "3.12"
     steps:
```

Comment:
As mentioned elsewhere, we unfortunately cannot drop support for Python 3.8 at this time.